kolibri/plugins/learn/assets/src/state/actions/main.js

```diff
@@ -297,9 +297,7 @@ function triggerSearch(store, channelId, searchTerm) {
     return;
   }
 
-  const contentCollection = ContentNodeResource.getPagedCollection(
```

Comment:
Is this from prettify linting? A bit confused why this is happening in files you haven't otherwise touched.

Reply:
Odd - not sure how this got into develop with it failing to pass the prettify linting in the first place. Most confusing.